src/System.Management.Automation/namespaces/ProviderBase.cs

```diff
@@ -1384,6 +1385,21 @@ public PSHost Host
             }
         }
 
+        /// <summary>
+        /// Gets the default item separator character for this provider.
+        /// </summary>
+        public virtual char ItemSeparator => Path.DirectorySeparatorChar;
+
+        /// <summary>
+        /// Gets the alternate item separator character for this provider.
+        /// </summary>
+        public virtual char AltItemSeparator =>
+#if UNIX
+            Utils.Separators.Backslash[0];
```

Comment:
@daxian-dbw Why don't we use `Path.DirectorySeparatorChar`?
https://github.com/dotnet/corefx/blob/78c4c8bea5121408b0b420c2aa029cdd992a0b26/src/Common/src/CoreLib/System/IO/PathInternal.Unix.cs#L14